### PR TITLE
Changes for issue 333, man page edits from Facts section to end

### LINKED ISSUE #333: Rho man page: edits part 3, sections facts, scanning, etc. to end
The final round of man page changes will include the Facts, Scanning, Examples, etc., sections to the end of the document. It's about 4 printed pages in total, but with the long example facts list and the Authors and Copyright sections, it's about 3 pages max.

### PR DESCRIPTION
**Pull request by jlprevatt-zz**
Submitting changes for the issue 333 "part 3" edit of the man page.

Closes #333

### CODE REVIEW

**Comment by jlprevatt-zz** [comment]
Review is from Facts section to the end.

File: doc/source/man.rst [hunk 1]
```diff
@@ -250,54 +250,52 @@ As the network infrastructure changes, it might be necessary to delete some netw
...
 -----
 
-The ``fact`` command is used to understand information that can be reported or to alter the contents of a report created from the ``rho scan`` command.
+Use the ``rho fact`` command to view information that can be reported in a scan or to alter the contents of a report that is created from the ``rho scan`` command.
 
 Listing Facts
 ~~~~~~~~~~~~~
 
-A list of facts that can be gathered during the scanning process can be obtained with the ``list`` command.
+To generate a list of facts that can be gathered during the discovery scanning process, use the ``rho fact list`` command.
```
Comment (chambridge) [inline doc/source/man.rst]: should we be consistent an say discovery and inspection scanning process?

File: doc/source/man.rst [hunk 2]
```diff
@@ -250,54 +250,52 @@ As the network infrastructure changes, it might be necessary to delete some netw
...
-  The path of the comma-separated values (CSV) file to be written.
+  Contains the path of the comma-separated values (CSV) report file to be written as output. Creates a new report with the sensitive facts encrypted.
 
 Scanning
 --------
 
-The ``scan`` command is the one that actually runs discovery on the network. This command scans all of the servers within the range, and then writes the information to a CSV file.
-
-A scan can be run by specifying the profile to use and where to write the CSV file:
+Use the ``rho scan`` command to run discovery scans on the network. This command scans all of the host names or IP addresses that are defined in the supplied network profile, and then writes the report information to a comma separated values (CSV) file.
```
Comment (chambridge) [inline doc/source/man.rst]: Again "discovery and inspection scans"?

File: doc/source/man.rst [hunk 3]
```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
...
 
 ``--cache``
 
-  This argument can be used if a profile has previously been used for a discovery and nothing new needs to be found during the scan
+  Restricts the scope of the scan to the hosts that were discovered in the previous scan. Use this option to discover software on hosts that were discovered in a previous scan. Do not use this option to scan for new hosts.
 
 ``--vault=vault_file``
 
-  This contains the path of the file that contains the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
```
Comment (kdelee) [inline doc/source/man.rst:336]: `s/Rho/rho/`
I think we are sticking with lowercase `rho` : [see this pr](https://github.com/quipucords/rho/pull/264/files)

File: doc/source/man.rst [hunk 4]
```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
...
 ``--ansible-forks=num_forks``
 
-  This value is used to determine the number of systems to scan in parallel. The current default is 50 concurrent connections.
+  Sets the number of systems to scan in parallel. The default number is 50 concurrent connections.
 
 Options for All Commands
 ------------------------
 
-A the following option is allowed with every command for rho.
+The following options are available for every Rho command.
```
Comment (kdelee) [inline doc/source/man.rst:349]: `s/Rho/rho/` again -- if we are sticking with `rho`

File: doc/source/man.rst [hunk 5]
```diff
@@ -327,52 +325,52 @@ A scan can be run by specifying the profile to use and where to write the CSV fi
...
+The authentication profile credentials that are used to access servers are stored with the network profile configuration in an AES-256 encrypted configuration file. A vault password is used to access this file. The vault password and decrypted file contents are in the system memory, and could theoretically be written to disk if memory swapping is enabled.
 
-While the vault password can be passed via a file to run ``rho`` without prompts (such as scheduling a cron job), using this can be risky and should be stored in a location with limited access; be cautious about using this mechanism.
+Although you can run the ``rho`` command without prompts (such as scheduling a cron job) by using a file to pass the vault password, the use of a file for vault password storage is not without risk; therefore, its use requires caution. The vault password allows access to encrypted Rho data that could contain sensitive information. Make sure that this vault password file, if used, is stored in a location that has limited access.
```
Comment (kdelee) [inline doc/source/man.rst:373]: `s/Rho/rho/` again

**Review by kdelee — COMMENTED** [review]
LGTM except the `rho` vs `Rho` issue. 
It seems to me we are sticking with lowercase, but if anyone else has a different take, feel free to chime in.

**Review by elyezer — COMMENTED** [review]
Changes look good. Please consider @chambridge comments.

About @kdelee comments (Rho vs rho) I think we decided to use Rho but this should be double checked and ensured to be consistent. I don't have a personal preference here.

File: doc/source/man.rst (hunk 3 above)
Comment (jlprevatt-zz) [reply doc/source/man.rst:336]: Thanks for pointing me to that pr. Rho is a tool; rho is a command. So I had an existential crisis regarding referring to Rho or rho for different sentences, and I changed my mind about what to do several times (here's a couple of examples of references to other things that are both tools and commands: http://docs.ansible.com/ansible/latest/ansible-playbook.html and https://git-scm.com/docs/git). If everyone prefers that I stick with rho in all cases to match other Rho internal/developer doc, then I can try to do it in this man page doc. That argument would not win for external/customer doc, though.

Comment (chambridge) [reply doc/source/man.rst:336]: I leave it to you @jlprevatt. I've leaned to lowercase `rho`, as I dont' really separate the tool and the command, but I just wanna ship it. 😄

Comment (kdelee) [reply doc/source/man.rst:336]: I agree, you can decide.

Comment (jlprevatt-zz) [reply doc/source/man.rst:336]: Actually, I just paged through the entire doc searching for "Rho" and "rho." I think all of the references to "Rho" were accurate in the sense of talking about "this is what the tool does" vs. "this is what the command does." I changed a few references to "the rho command" in paragraphs to include the backwards single quotation marks around "rho" to add highlighting. 

The only thing I didn't touch was the way things are formatted in the Authors section--I'll leave that as it was originally formatted by the creators/contributors.

So I rescind part of my former reply, and just say my existential crisis came back.

File: doc/source/man.rst (hunk 1 above)
Comment (jlprevatt-zz) [reply doc/source/man.rst]: Change made.
I did a global search for "discovery" and added an "and inspection" in locations where it was missing.

File: doc/source/man.rst (hunk 2 above)
Comment (jlprevatt-zz) [reply doc/source/man.rst]: Change made.
I did a global search for "discovery" and added an "and inspection" in locations where it was missing.

File: doc/source/man.rst (hunk 4 above)
Comment (jlprevatt-zz) [reply doc/source/man.rst:349]: See your first comment about Rho/rho, existential crisis, etc.

File: doc/source/man.rst (hunk 5 above)
Comment (jlprevatt-zz) [reply doc/source/man.rst:373]: Thank you for making these comments, even though I decided I disagreed a bit. It made me go back and check all the references.

File: doc/source/man.rst (hunk 3 above)
Comment (kdelee) [reply doc/source/man.rst:336]: I appreciate your struggle 😣 Thanks

**Review by chambrid — APPROVED** [review]

**Pull request merged**